### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by srijani07**
image

### CODE REVIEW

File: cart.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.cart-wrapper {
+  display: flex;
+  gap: 2rem;
+  justify-content: center;
+}
+
+@media (max-width: 800px) {
+  .cart-wrapper {
```
Comment (HarshnaNHaswani) [inline cart.css:18]: use ems

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        
+
+
+
+
+
+
+
+</body>
+
```
Comment (HarshnaNHaswani) [inline cart.html:91]: remove excess spacing

**Review by HarshnaNHaswani — APPROVED** [review]
Good to merge

File: cart.html (hunk 2 above)
Comment (srijani07) [reply cart.html]: Done. Thanks!

**Pull request merged**